WorldBank-Transport/DRIVER
# Add sample data for public users

## Overview
In order to help users get a working environment we want to provide sample data that can be used to populate the system. For simplicity this is done as statically defined sample data that can be used with the same import process that real data can be imported by.

### Checklist


### Demo
Data is anonymized from a real-world data set, with identifying information removed, dates randomly assigned, and locations adjusted according to a random normal distribution. This means that records generally follow a realistic geographic distribution but individual points do not necessarily coincide with a road and are not constrained within a fixed radius of their originating location:
![screen shot 2018-09-19 at 4 15 05 pm](https://user-images.githubusercontent.com/1032849/45778730-66d85480-bc27-11e8-851b-d948e4c28e73.png)
![screen shot 2018-09-19 at 4 15 32 pm](https://user-images.githubusercontent.com/1032849/45778732-68a21800-bc27-11e8-8d23-96e800892400.png)

### Notes
Currently, a couple caveats apply:
- The static sample data has a occurrence date directly associated with each record, and so as time progresses this file will become stale.
- The import process creates a new Record Type for the records it imports each time it is run, making it less desirable for use as a refresh process when records begin to cross the 3-month default occurrence window.

## Testing Instructions
- Obtain an authorization token
- Run (You likely will need to install `python-dateutil`, `pytz` and `gdal`, potentially other dependencies as well):
```bash
python scripts/load_incidents_v3.py --authz 'Token [API Token]' scripts/sample_data/
```
- You should have the sample data available 

Closes [PT160318086](https://www.pivotaltracker.com/story/show/160318086)
